replication/logic/replication_logic.go

```diff
@@ -396,9 +396,7 @@ func (fs *Filesystem) doPlanning(ctx context.Context) ([]*Step, error) {
 		switch fs.policy.EncryptedSend {
 		case True:
 			encryptionMatches = resumeToken.RawOK && resumeToken.CompressOK
-		case False:
-			encryptionMatches = !resumeToken.RawOK && !resumeToken.CompressOK
-		case DontCare:
+		case False, DontCare:
```

Comment:
@Infinisil any particular reason for this change here? I think we always set `policy.EncryptedSend` to `True` or `False` at this time, maybe we should instead expose `DontCare` in the config?